_proto/hapi/services/tiller.proto

```diff
@@ -69,16 +69,33 @@ service ReleaseService {
 }
 
 //
-// ListReleasesRequest:
+// ListReleasesRequest is a request for a list of releases.
 //
-//		TODO
+// Releases can be retrieved in chunks by setting limit and offset.
 //
+// Releases can be sorted according to a few pre-determined sort stategies.
 message ListReleasesRequest {
-	// The maximum number of releases to be returned
+	// Limit is the maximum number of releases to be returned.
 	int64 limit  = 1;
-	
-	// The zero-based offset at which the returned release list begins 
+
+	// Offset is the zero-based offset at which the returned release list begins.
```

Comment:
Rather than have an index here, it might be better to provide a way to specify the starting 'key', if it was only say names, you could offer pagination that way, rather than have to keep track off the offset. Just a thought.